native/metrodroid/Settings.bundle/de.lproj/Root.strings

```diff
@@ -0,0 +1,2 @@
+/* This file is autogenerated. Do not edit manually.  */
```

Comment:
This, and all the other strings appear to be automatically generated as part of the build process, so I think these can be removed (07c37f28).

Reply:
OK